packages/lib/defaultEvents.ts

```diff
@@ -177,14 +178,8 @@ export const getUsernameList = (users: string | string[] | undefined): string[]
   // So, even though this code handles even if individual user is dynamic link, that isn't a possibility right now.
   users = arrayCast(users);
 
-  const allUsers = users.map((user) =>
-    user
```

Comment:
this looks like it was wrong, if you lowercase and replace " " with + before splitting you end up splitting too late. Split should happen first before spaces are changed into their replacement character;